internal/controller/employee.go

```diff
@@ -132,15 +152,23 @@ func (c *EmployeesController) UpdateEmployee(w http.ResponseWriter, r *http.Requ
 	w.Write(response)
 }
 
-func (e *EmployeesController) GetAllEmployees(w http.ResponseWriter, r *http.Request) {
+func (c *EmployeesController) GetAllEmployees(w http.ResponseWriter, r *http.Request) {
 	if r.Method != http.MethodGet {
 		errorHandler(w, r, &HTTPError{Detail: "invalid method at get all employees", Status: http.StatusMethodNotAllowed})
 		return
 	}
 
-	employees, err := e.Repo.GetAll(r.Context())
+	page, _ := strconv.ParseInt(r.URL.Query().Get("page"), 10, 64)
+	pageSize, _ := strconv.ParseInt(r.URL.Query().Get("size"), 10, 64)
+
+	if page <= 0 || pageSize <= 0 {
```

Comment:
That's not entirely correct, because in case you have pageSize set, but not page number it will set both parameters to defaults.

Reply:
fixed